src/temp/SiteDropdown/SiteDropdown.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react';
+import SiteIcon from '../icons/SiteIcon';
+import styles from './SiteDropdown.module.scss';
+import DropdownArrow from '../icons/DropdownArrow';
+
+interface SiteType {
+  name: string;
+  area: Number;
+}
+interface Props {
+  siteList: SiteType[];
+  selectedOption: SiteType;
+  isOpen: boolean;
+}
+
+const SiteDropdown = ({ selectedOption, siteList, isOpen }: Props) => {
```

Comment:
could you please rename component  to ProjectSiteDropdown ? and also use suitable name for interface